conda-forge.yml

```
@@ -1,9 +1,15 @@
-build_platform: {osx_arm64: osx_64}
+build_platform:
+  osx_arm64: osx_64
+  win_arm64: win_64
 conda_forge_output_validation: true
-provider: {linux_aarch64: azure, linux_ppc64le: azure, win: azure}
 github:
   branch_name: main
   tooling_branch_name: main
 conda_build:
   pkg_format: '2'
+provider:
+  linux_aarch64: azure
+  linux_ppc64le: azure
+  win-arm64: azure
 test: native_and_emulated
+conda_build_tool: rattler-build
```

I tend to migrate both at the same time (for more speed)
```suggestion
conda_build_tool: rattler-build
conda_install_tool: pixi
```

---

Me too but we ran into: https://github.com/conda-forge/ninja-feedstock/pull/38#issuecomment-2827915820

---

I think this can be fixed if we cross-compile for ppc64le and aarch64 from linux-64. Would also drastically speed up compilation for those platforms. Want me to quickly give that a try?

---

Yes, please. I would have assumed that we already do it.

---

Done in c75d4ce

---

Seems to work, builds are now even faster across the board. `linux-aarch64` now only takes 4 minutes instead of 40. `linux-64` now takes 1 minute 30 seconds. Pretty crazy improvements.